### PR TITLE
Run headless browser tests in CI

### PR DESCRIPTION
**Pull request by fitzgen**
@rylev or @alexcrichton want to take a look at this?

### CODE REVIEW

**Comment by fitzgen** [comment]
Hmm... why isn't CI getting triggered?

**Comment by alexcrichton** [comment]
Oh [looks like it did](https://dev.azure.com/rustwasm/gloo/_build/results?buildId=279&view=results) but maybe it's a bug in azure pipelines that the status there wasn't reported back here?

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -10,6 +11,27 @@ jobs:
           rust_version: stable
       - bash: |
           export RUST_BACKTRACE=1
-          cargo check --all || exit
           cargo test --all || exit
-        displayName: Run cargo check and test 
+        displayName: Run cargo check and test
```
Comment (alexcrichton) [inline .azure-pipelines.yml]: Was there maybe some whitespace changes here?

Comment (fitzgen) [reply .azure-pipelines.yml]: Just removing a trailing space -- I think the issue is `displayName` coming before `script`? Trying to swap the order...

**Comment by fitzgen** [comment]
Phew! Ok 5 crappy "fixing CI (again)" style commits and now our headless browser tests are running in CI!

**Pull request merged**